packages/liveblocks-react/src/umbrella-store.ts

```diff
@@ -1022,6 +1129,181 @@ export class UmbrellaStore<M extends BaseMetadata> {
   public setQuery4Error(queryKey: string, error: Error): void {
     this.setQuery4State(queryKey, { isLoading: false, error });
   }
+
+  public loadThreadsAndNotifications(
+    roomId: string,
+    options: GetThreadsOptions<BaseMetadata>,
+    queryKey: string
+  ): Promise<ThreadData[]> {
```

Comment:
Maybe, in future, we could not require the `queryKey` so that `useThreads` and others to only pass the `roomId` and `options` as that is enough to generate the query key.